protostar-rust/src/lib.rs

```diff
@@ -82,3 +91,174 @@ fn run_tests_in_file(
     }
     Ok(())
 }
+
+pub fn protostar_config_for_package(
+    metadata: &Metadata,
+    package: &PackageId,
+) -> Result<ProtostarTestConfig> {
+    let raw_metadata = metadata
+        .get_package(package)
+        .ok_or_else(|| anyhow!("Failed to find metadata for package = {package}"))?
+        .tool_metadata("protostar")
+        .ok_or_else(|| anyhow!("Failed to find protostar config for package = {package}"))?
+        .clone();
```

Comment:
Shouldn't we just log this part? I don't feel like it is necessary with minimal setup to have any arguments regarding protostar in Scarb.toml

Reply:
Fair point, but let's change this logic in the separate PR.

Reply:
#2041